tests/rbd_mirror/test_rbd_mirror_snapshot.py

```diff
@@ -26,9 +26,9 @@ def run(**kw):
         poolname = mirror1.random_string() + "_tier_1_rbd_mirror_pool"
         imagename = mirror1.random_string() + "_tier_1_rbd_mirror_image"
         imagespec = poolname + "/" + imagename
-
         mirror1.initial_mirror_config(
-            mirror2,
+            mirror1=mirror1,
```

Comment:
added these config changes becasue mirror1 and mirror2 becomes positional arguments in the definition call hence added,
Failure without adding:
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-O1NDZG/test_rbd_mirror_snapshot_pool_0.log
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-ZECGTZ/test_rbd_mirror_snapshot_cluster_0.log

Reply:
@sunilangadi2 In the latest code for initial_mirror_config, mirror1 is not positional argument. https://github.com/red-hat-storage/cephci/blob/master/tests/rbd_mirror/rbd_mirror_utils.py#L218 Could you please change your code accordingly?

Reply:
@manasagowri `self` is taken as `mirror1`  if i don't assign here I am facing the failures as mentioned logs

Reply:
Thanks, @manasagowri Reverted the changes here.